lib/resources/dstu2/soarian_medication_statement.yaml

```diff
@@ -0,0 +1,202 @@
+---
+name: MedicationStatement
+field_name_base_url: http://hl7.org/fhir/DSTU2/medicationstatement-definitions.html#MedicationStatement
+fields:
+- name: resourceType
+  required: 'Yes'
+  cardinality: 1..1
+  type: string
```

Comment:
remove all fields that don't have associated bindings. Likely, this file gets removed completely and a contained medication yaml is created instead, with just the medication.code field bound.